book/src/slashing.md

```diff
@@ -0,0 +1,57 @@
+# Slashing
+
+Proof of Stake consensus requires that byzantine behavior be punished by loss of stake and rewards, also known as slashing. 
+Solana implements slashing in 2 regimes: the validator regime and the replicator regime, corresponding to the two rewards regimes.
+During slashing, the network destroys some "penalty percentage" of lamports staked/earned in the rewards-generating accounts.
+
+## Validator regime
+
+Validators cast their votes to a Vote account to which stake has been [passively delegated](stake-delegation-and-rewards.md).
+The Vote program keeps track of rewards for all of the stakes, which allows it to be a central point for slashing.
+The basic mechanism of slashing is:
+
+1. a proof of byzantine behavior is submitted to the Vote program
```

Comment:
@garious would barf on the `byzantine`

Reply:
Nah, I've come around to it. Just capitalize it please.